commit 0bc364615b5b2f0098a0d239fa0c0506bc4db9b6
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Thu Mar 23 17:05:26 2023 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Mar 23 14:05:26 2023 -0700

    Qa issues (#1568)
    
    * QA issues.
    
    * Refactored.
    
    * Removed commented out lines.
    
    * Changed to use logger.
    
    ---------
    
    Co-authored-by: Yuan-Ting Hsieh (謝沅廷) <yuantingh@nvidia.com>
---
 nvflare/private/fed/server/job_runner.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/job_runner.py b/nvflare/private/fed/server/job_runner.py
index 114a4748..98248a41 100644
--- a/nvflare/private/fed/server/job_runner.py
+++ b/nvflare/private/fed/server/job_runner.py
@@ -483,9 +483,10 @@ class JobRunner(FLComponent):
             with self.lock:
                 del self.running_jobs[job_id]
             self.fire_event(EventType.JOB_ABORTED, fl_ctx)
+            return ""
         else:
             self.log_error(fl_ctx, f"Job {job_id} is not running. It can not be stopped.")
-            raise RuntimeError(f"Job {job_id} is not running.")
+            return f"Job {job_id} is not running."
 
     def stop_all_runs(self, fl_ctx: FLContext):
         engine = fl_ctx.get_engine()
